docs/compatibility/compatibility-matrix.rst

```diff
@@ -13,7 +13,7 @@ You can also refer to the :ref:`past versions of ROCm compatibility matrix<past-
 
 .. note::
 
-   AMD Radeon and Radeon Pro GPUs can be used for graphics-related purposes or in headless environments for compute purposes (no graphics components). If you’re not using Radeon or Radeon Pro GPUs in a headless environment, review the `Compatibility matrices <https://rocm.docs.amd.com/projects/radeon/en/latest/docs/compatibility.html>`_ in the Use ROCm on Radeon GPU documentation.
+   Like AMD Instinct accelerators, AMD Radeon and Radeon Pro GPUs can be used in environments for compute purposes (no display information or graphics). If using AMD Radeon or Radeon Pro GPUs with ROCm for graphics-related purposes (for example, display connected), review the `Compatibility matrices <https://rocm.docs.amd.com/projects/radeon/en/latest/docs/compatibility.html>`_ in the Use ROCm on Radeon GPU documentation to confirm system requirements.
```

Comment:
```suggestion
   ROCm supports some AMD Radeon and Radeon Pro GPUs. If using ROCm with these GPUs for graphics workloads, check the `Use ROCm on Radeon GPU compatibility documentation <https://rocm.docs.amd.com/projects/radeon/en/latest/docs/compatibility>`_ to verify compatibility and system requirements.
```
Something like this might be simpler IMO

Reply:
Maybe we could add this as a footnote for RDNA3, RDNA2, gfx1100, and gfx1030 instead? That look cluttered though.